src/components/Pega_Extensions_ImageMagnify/demo.stories.tsx

```diff
@@ -0,0 +1,58 @@
+// @ts-nocheck
+import { withKnobs } from '@storybook/addon-knobs';
```

Comment:
knobs are deprecated, we're using "controls". Please refer other stories files to provide editable story arguments.